azurerm/internal/services/sentinel/sentinel_alert_rule_scheduled_resource.go

```diff
@@ -109,6 +109,7 @@ func resourceSentinelAlertRuleScheduled() *schema.Resource {
 						string(securityinsight.AttackTacticLateralMovement),
 						string(securityinsight.AttackTacticPersistence),
 						string(securityinsight.AttackTacticPrivilegeEscalation),
+						string(securityinsight.PreAttack),
```

Comment:
```suggestion
						string(securityinsight.AttackTacticPreAttack),
```

Reply:
I was to hasty to submit the PR. I think this should be fixed now.